config/environments/production.rb

```diff
@@ -95,12 +95,13 @@
   # MAILER
   config.action_mailer.default_url_options = { :host => ENV["HOST"] }
   ActionMailer::Base.smtp_settings = {
-    :user_name => ENV["SENDGRID_USERNAME"],
-    :password => ENV["SENDGRID_PASSWORD"],
-    :domain => ENV["HOST"],
-    :address => 'smtp.sendgrid.net',
-    :port => 587,
-    :authentication => :plain,
-    :enable_starttls_auto => true
-  }  
+    password: Figaro.env.sendgrid_pwd,
```

Comment:
Nope. Heroku will set them to "SENDGRID_USERNAME". Name them this way in figaro.

Reply:
Thanks ! 👍